connector_redmine/models/redmine_backend.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from openerp.exceptions import Warning
+from openerp.tools.translate import _
+from openerp.addons.connector.connector import ConnectorEnvironment
+from ..session import RedmineConnectorSession
```

Comment:
I'm not using any python library here. What is there to protect ?
Also, if an error happens, I don't want it to go silently. I prefer the exception to be raised when starting the server instead of having surprises later.